### DISCUSSION TITLE
Control of "Stay Over Guests" in Sims 4

### DISCUSSION THREAD

**Discussion by ladynimue** [TitanNano/TS4ControlAnySim#70] [Ideas]
Are there any plans to add control for "Stay Over Guests"? This is a new feature since the Growing Together pack and Control Any Sim only adds the sim's grayed out icon to the panel but doesn't let you select them. This would be a great addition if you could make it happen.

**Comment by TitanNano** [comment]
I would be happy to add the feature, but I do not have the capacity to work on it at the moment.

**Reply by TitanNano** [reply]
There is a feature request ticket: https://github.com/TitanNano/TS4ControlAnySim/issues/63

**Comment by TitanNano** [comment]
this has been fixed in release [v1.2.5](https://github.com/TitanNano/TS4ControlAnySim/releases/tag/v1.2.5) and later.
